Source/Collection/Plugins/PluginAction/CollectionScrollViewDelegateProxyPlugin.swift

```diff
@@ -71,3 +71,48 @@ public extension BaseCollectionPlugin {
     }
 
 }
+
+/// Proxy of  `CollectionScrollViewDelegateProxyPlugin` events
+#if os(iOS)
+@available(iOS 13.0, *)
+public extension CollectionScrollViewDelegateProxyPlugin {
+
+    // MARK: - Private static properties
+
+    private static var didScrollCompositionLayoutSection = BaseEvent<ItemsInvalidationResult>()
+
+    // MARK: Nested types
+
+    typealias ItemsInvalidationResult = (items: [NSCollectionLayoutVisibleItem], offset: CGPoint, environment: NSCollectionLayoutEnvironment)
```

Comment:
```suggestion
    public typealias ItemsInvalidationResult = (items: [NSCollectionLayoutVisibleItem], offset: CGPoint, environment: NSCollectionLayoutEnvironment)
```
и старый алиас в контроллере тогда удалить можно будет.